src/main/java/net/neoforged/neoforge/common/Tags.java

```diff
@@ -887,10 +901,22 @@ public static class Biomes {
         public static final TagKey<Biome> IS_COLD_NETHER = tag("is_cold/nether");
         public static final TagKey<Biome> IS_COLD_END = tag("is_cold/end");
 
+        public static final TagKey<Biome> IS_TEMPERATE = tag("is_temperate");
+        public static final TagKey<Biome> IS_TEMPERATE_OVERWORLD = tag("is_temperate/overworld");
+        public static final TagKey<Biome> IS_TEMPERATE_NETHER = tag("is_temperate/nether");
+        public static final TagKey<Biome> IS_TEMPERATE_END = tag("is_temperate/end");
```

Comment:
I would make the argument that all new tags added to Neo should have javadocs. What behaviors are enforced by adding a biome to `is_temperate`? What behaviors are not enforced?